stats/nba_sql.py

```diff
@@ -193,20 +235,27 @@ def do_create_schema(object_list):
     for obj in object_list:
         obj.create_ddl()
 
-def populate_base_tables(seasons, request_gap, team_requester, player_requester, event_message_type_builder):
+
+def populate_base_tables(
```

Comment:
Btw, is this a convention, you want to adopt? i.e, every parameter on a different line? If you set `max-line-length` to 120 chars, then you wouldn't see flake8 complain.

Reply:
I'd prefer not to. I think 120 is good, I will go back and make these less insane. Was just trying to get it to a baseline.